### PR TITLE
Would be nice to know when the etcdbackup finishes

### PR DESCRIPTION
**Pull request by thekad**
```release-note
NONE
```

### CODE REVIEW

File: pkg/entrypoint/etcdbackup/etcdbackup.go [hunk 1]
```diff
@@ -94,5 +94,8 @@ func start(cfg *cmdConfig) error {
 	default:
 		flag.Usage()
 	}
+
+	log.Print("etcdbackup finished")
```
Comment (olga-mir) [inline pkg/entrypoint/etcdbackup/etcdbackup.go]: just out of curiosity is there any rule where we use `log.Info`[f] and where `log.Print`, I can see we have both, but "Print" not as much as "info"

Comment (kwoodson) [reply pkg/entrypoint/etcdbackup/etcdbackup.go]: I think I'd prefer `.Info` as I do know that logging generally has a switch to turn on logging levels and `Info` falls into a specific level whereas I don't know what `Print` does.

Comment (thekad) [reply pkg/entrypoint/etcdbackup/etcdbackup.go]: Yeah I only copy/pasted the line from a few lines above where the "starting" message appears. I do agree it is confusing, I'll change it to Info.

**Comment by thekad** [comment]
Couple of timeout errors
/retest

**Pull request merged**